src/Components/CreateImageWizard/steps/FileSystem/components/FileSystemTable.tsx

```diff
@@ -153,6 +153,10 @@ const FileSystemTable = () => {
     });
   };
 
+  const rootPartitionsCount = partitions.filter(
```

Comment:
it runs on every render, even when the partitions array hasn't changed, wdyt to maybe change it to 
```
const rootPartitionsCount = useMemo(
  () => partitions.filter((p) => p.mountpoint === '/').length,
  [partitions]
);
```

Reply:
That's a good point, thanks! I added the useMemo:)